src/components/GraphiQL.js

```diff
@@ -476,7 +476,12 @@ export class GraphiQL extends React.Component {
   _fetchSchema() {
     const fetcher = this.props.fetcher;
 
-    const fetch = observableToPromise(fetcher({ query: introspectionQuery }));
+    const fetch = observableToPromise(
+      fetcher({
+        query: introspectionQuery,
+        operationName: 'IntrospectionQuery',
```

Comment:
I think it's dangerous to hard-code the name of introspection query here.
Especially since `introspectionQuery` is imported from `graphql-js` which is one of `peerDependencies`.

It's make sense to add something like this into [introspectionQueries.js](https://github.com/graphql/graphiql/blob/master/src/utility/introspectionQueries.js).
```js
export const introspectionQueryName = getOperationAST(introspectionQuery).name.value;
```

Reply:
Ok, I'll update the PR.